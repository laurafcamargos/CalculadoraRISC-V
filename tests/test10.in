5
/
2
*
7
/
3
f
