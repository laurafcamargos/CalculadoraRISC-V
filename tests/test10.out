Digite um número: Digite a operação (+, -, *, /, u, f): Digite um número: Resultado: 2
Digite a operação (+, -, *, /, u, f): Digite um número: Resultado: 14
Digite a operação (+, -, *, /, u, f): Digite um número: Resultado: 4
Digite a operação (+, -, *, /, u, f): 
